server/index.js

```diff
@@ -518,22 +529,28 @@ apiRouter.post('/app/:application/version/:version/current', function(req, res)
 	});
 });
 
-const githubRouter = express.Router();
-// Callback for GitHub logins
-// Note that we reject any user here that is not in the Collaborne organization; maybe this should be done better.
-githubRouter.get('/oauth', function(req, res) {
+function handleOAuthCallback(tokenUrl, clientId, clientSecret, code, callback) {
```

Comment:
TODO: separate this one out, it's just a refactoring to be able to reuse the code.